mappings/net/minecraft/world/level/storage/LevelStorage.mapping

```diff
@@ -5,6 +5,7 @@ CLASS net/minecraft/class_32 net/minecraft/world/level/storage/LevelStorage
 	FIELD field_17668 dataFixer Lcom/mojang/datafixers/DataFixer;
 	FIELD field_200 TIME_FORMATTER Ljava/time/format/DateTimeFormatter;
 	FIELD field_25020 GENERATOR_OPTION_KEYS Lcom/google/common/collect/ImmutableList;
+	FIELD field_31836 DEFAULT_ICON Ljava/lang/String;
```

Comment:
Is this really something default?

Reply:
I don't understand what you mean, yes, this is the default `icon.png` for displaying on the Select World screen when no icon for that world is present